ansible/roles/machine/setup/tasks/install_packages.yml

```diff
@@ -4,6 +4,15 @@
     name: '*'
     state: latest
 
+# temporary workaround for rawhide issue
+- name: remove packages that cause conflicts on rawhide
+  dnf:
+    name:
+      - gdbm
+      - python2-chardet
+    state: absent
+  when: fedora_version == 'rawhide'
```

Comment:
Just rawhide or also Fedora 30?

Reply:
Apparently, this is not required any more.